sdks/python/.pylintrc

```diff
@@ -138,7 +137,6 @@ disable =
   simplifiable-if-statement,
   stop-iteration-return,
   super-init-not-called,
-  super-on-old-class,
```

Comment:
Did some wires get crossed here?

The PR title says this fixes undefined-variable and unexpected-keyword-arg, but those aren't being enabled, and some other checks are enabled.

Reply:
Yes, you are right  ,sorry,  I forgot change the PR name for a proper one. 
I change the name of the PR to reflect the changes that were made. 
Thanks @TheNeuralBit

Reply:
Got it, thanks!